src/domain/metric/service/postMetric.js

```diff
@@ -30,7 +31,7 @@ export default (posts: Post[]): PostMetric[] => {
     return {
       post,
       id: post.id,
-      createdTime: new Date(post.created_time),
+      createdTime: moment(post.created_time).format(),
```

Comment:
@reyhansofian so we must `format` on the file that consumes this metric, not on the exposed metric itself.

OR

if the `created_time` itself is the root cause, it must be handled inside this metric, and still exposed as `Date`. We can't exposed `moment` standard, it will broke other files for sure.